crates/ruff/src/codes.rs

```diff
@@ -701,6 +701,8 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<Rule> {
         (Ruff, "006") => Rule::AsyncioDanglingTask,
         (Ruff, "007") => Rule::PairwiseOverZipped,
         (Ruff, "100") => Rule::UnusedNOQA,
+        (Ruff, "200") => Rule::MutableDataclassDefault,
```

Comment:
I think we should just make these `RUF008` and `RUF009`. The choice of codes within the `RUF` category doesn't have semantic meaning, so I'd rather stick to autoincrement until the point at which we make an explicit decision to do otherwise.

Reply:
Makes sense to me. Actually I found it to be quite difficult to choose something right. Might just be because the "new checks" are extremely similar to the bugbear ones and one could certainly make a case for them just being the same.

In the end I thought that matching the bugbear implementation and creating new ones is better. But at some point they probably should be combined (imho).